include/libu64/pad.h

```diff
@@ -0,0 +1,22 @@
+#ifndef PAD_H
+#define PAD_H
+
+#include "types.h"
+#include "libultra/osContPad.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct {
+  OSContPad now;
+  OSContPad last;
+  OSContPad on;
+  OSContPad off;
```

Comment:
prev/press/rel would probably be clearer if this is the same as z64's `Input` struct.